src/components/charts/SummaryChartGroup.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import SummaryPieChart from './SummaryPieChart';
+import SummaryBarChart from './SummaryBarChart';
+import './SummaryChartGroup.less';
+
+class SummaryChartGroup extends Component {
+  render() {
+    const width = (typeof this.props.width === 'number') ? `${this.props.width}px` : this.props.width;
+    return (
+      <div className="summary-chart-group" style={{ width }}>
+        {
+          this.props.summaries.map((item, index) => (
+            <div className="summary-chart-group__column" key={item.title}>
+              {
+                index > 0 && <div className="summary-chart-group__column-left-border" />
+              }
+              {
+                item.type === 'pie'
+                  ? (
+                    <SummaryPieChart
+                      data={item.data}
+                      title={item.title}
+                      localTheme={this.props.localTheme}
+                    />
+                  ) : (
+                    <SummaryBarChart
+                      data={item.data}
+                      title={item.title}
+                      localTheme={this.props.localTheme}
+                      vertical
```

Comment:
do these props need to be set to something? ie. `vertical={true}`?

Reply:
I agree with you! Just Codacy told me`boolean attribute` something and asked me to change into this...